fastlane/Fastfile

```diff
@@ -122,6 +163,19 @@ platform :ios do
   lane :clean do
     clean_testflight_testers
   end
+
+  desc "`fastlane ios upload-diawi` - upload .ipa to diawi"
+  desc "also notifies in a GitHub comments and in Slack #jenkins channel"
+  desc "expects to have an .ipa prepared: `status-adhoc/StatusIm.ipa`"
+  desc "expects to have diawi token as DIAWI_TOKEN  env variable"
+  desc "expects to have diawi token as GITHUB_TOKEN env variable"
```

Comment:
`expects to have diawi token` is repeated 3 times

Reply:
ah, ok
I will fix it and merge after that then :)